hw/ip/otbn/doc/_index.md

```diff
@@ -557,6 +557,15 @@ This is to allow OTBN applications to store sensitive information in the other h
 Each memory write through the register interface updates a checksum.
 See the [Memory Load Integrity]({{< relref "#mem-load-integrity" >}}) section for more details.
 
+### Instruction Prefetch
+
+OTBN employs an instruction prefetch stage which allows register file inputs to be taken directly from registers by pre-decoding instructions in the prefetch stage.
+This is required to enable the blanking SCA hardening measure (NOT YET IMPLEMENTED OR DOCUMENTED).
```

Comment:
Can you add a TODO here, just to make sure we don't miss it later? Maybe just "(TODO: Not yet implemented or documented)", so that it comes up with grep.